### DISCUSSION TITLE
Recommended RAM - Use along with blue lab

### DISCUSSION THREAD

**Discussion by emaragkos** [Marshall-Hallenbeck/red_team_attack_lab#14] [Q&A] [answered]
Hey there! Awesome work, thanks for your contribution! I have two quick questions.
1) What would be the recommended RAM to run all hosts smoothly?
2) Have you tried to use this along with DetectionLab/AttackRange/SecuityOnion or any other similar soluton? Any recommendation? I'm asking this because I love the fact that this focuses on the red perspective since I need a vulnerable environment for experimentation with TTPs, but still I need to be able to do threat hunting and detection engineering afterwards. 
Thanks :)

**Comment by Marshall-Hallenbeck** [comment, marked as answer]
Hey, sorry for the delay in the response here, been busy with the holidays and end of year stuff.

To answer your first question, I ran everything on a host with 16gigs of RAM... I'm not sure what the minimal RAM would be, but you can always change how much each host uses if you think it's too much!

To answer the second, this was actually based on AttackRange. If you look back in the commits it _was_ the attack range, then I slowly converted it into its completely own project. I felt like there were already great blue-team tools like DetectionLab & AttackRange, and I needed something Red Team centric. I pulled all the AttackRange stuff out because I didn't want to have extra dependencies and bloat, and also didn't want to steal their work or have licensing issues, but you could probably add it back in pretty easily since their work is just Vagrant & Ansible scripts.

**Reply by emaragkos** [reply]
Thanks for your feedback! Love the project, keep it up!
